pkg/config/machine/machine.go

```diff
@@ -27,17 +27,17 @@ const (
 
 // String returns the string representation of Type.
 func (t Type) String() string {
-	return [...]string{"Init", "ControlPlane", "Join"}[t]
+	return [...]string{"init", "controlplane", "worker"}[t]
 }
 
 // ParseType parses string constant as Type
 func ParseType(t string) (Type, error) {
 	switch t {
-	case "Init":
+	case "init":
 		return TypeInit, nil
-	case "ControlPlane":
+	case "controlplane":
 		return TypeControlPlane, nil
-	case "Join":
+	case "worker":
```

Comment:
This is now the source of truth for all things that use a machine's type in any way.